srv/modules/runners/benchmark.py

```diff
@@ -147,25 +163,70 @@ def run(**kwargs):
             # (when there is on ext_module) or an array :(
             # This needs a better solution...works only if benchmark.cfg is 2nd
             # entry in ext_modules
-            bench_dir = dirname(ext['stack'][1])
+            dir_options['bench_dir'] = os.path.dirname(ext['stack'][1])
+
+    return dir_options
 
-    default_collection = {}
-    with open('{}/collections/default.yml'.format(bench_dir), 'r') as yml:
+def __parse_collection(collection_file):
+    with open(collection_file, 'r') as yml:
         try:
-            default_collection = yaml.load(yml)
+            return yaml.load(yml)
         except YAMLError as error:
-            print('Error parsing default collection:')
-            print(error)
-            exit(1)
+            log.error('Error parsing collection {}:'.format(collection_file))
+            log.error(error)
+            raise error
+
+def rbd(**kwargs):
+    """
+    Run rbd benchmark job
+    """
+
+    client_glob = 'I@roles:client-rbd and I@cluster:ceph'
+    if 'client_glob' in kwargs:
+        client_glob = kwargs['client_glob']
+    log.info('client glob is {}'.format(client_glob))
+
+    dir_options = __parse_and_set_dirs(kwargs)
 
-    fio = Fio(bench_dir, work_dir, log_dir, job_dir)
+    default_collection = __parse_collection('{}/collections/default.yml'.format(dir_options['bench_dir']))
 
-    for job_spec in default_collection['jobs']:
+    fio = Fio(client_glob, 'rbd',
+            dir_options['bench_dir'],
+            dir_options['work_dir'],
+            dir_options['log_dir'],
+            dir_options['job_dir'])
+
+    for job_spec in default_collection['rbd']:
+        print(fio.run(job_spec))
+
+    return True
+
+def cephfs(**kwargs):
+    """
+    Run cephfs benchmark jobs
+    """
+
+    client_glob = 'I@roles:client-cephfs and I@cluster:ceph'
+    if 'client_glob' in kwargs:
```

Comment:
client_glob = kwargs.get('client_glob', None) 
will save you one line and allows you to set a default/fallback